src/utils/CookieExtractor.ts

```diff
@@ -0,0 +1,16 @@
+export default class CookieExtractor {
+
+    constructor(private readonly cookiesString: string) { }
+
+    public getCookie(name: string) {
```

Comment:
Shouldn't this function body be in constructor? Since the `cookiesString` don't change, it makes no sense to map the string to an object everytime you call this function.